saleor/graphql/checkout/types.py

```diff
@@ -233,6 +236,22 @@ class Meta:
         interfaces = [graphene.relay.Node, ObjectWithMetadata]
         filter_fields = ["token"]
 
+    @staticmethod
+    def resolve_shipping_address(root: models.Checkout, info):
+        return (
+            AddressByIdLoader(info.context).load(root.shipping_address_id)
+            if root.shipping_address_id
+            else None
+        )
```

Comment:
Isn't this way more readable?
```suggestion
        if not root.shipping_address_id:
            return
        return AddressByIdLoader(info.context).load(root.shipping_address_id)
```

Reply:
I removed it because this doesn't affect on performance.